cpp/src/barretenberg/stdlib/recursion/aggregation_state/aggregation_state.hpp

```diff
@@ -70,6 +70,8 @@ template <typename Curve> struct aggregation_state {
 
         auto* context = P0.get_context();
 
+        context->check_circuit();
```

Comment:
Do you want to leave this in? Currently it will only print information about failed gates

Reply:
Whoops, no, that should go